docs/index.md

```diff
@@ -0,0 +1,71 @@
+# Chisel
+
+**Chisel** is a tool for carving and cutting Debian packages.
+
+It can derive a minimal Ubuntu-like Linux distribution using a release
+database that defines "slices" of Debian packages. Slices enable users to
+cherry-pick just the files they need from the Ubuntu archives, and combine them
+to create a new root file system which can be packaged.
+
+If you need a way to create a minimal root file system based on Ubuntu
+packages, Chisel might be for you. It handles package slice dependency,
+fetching Ubuntu Pro packages, and allows you to create a fully operational but
+minimal root file system for your application.
+
+Chisel is particularly useful for developers who are creating minimal
+containers such as {{Rocks}}, or {{Docker}} images.
```

Comment:
Did you get this content from an existing place?

```suggestion
Chisel is able to extract a highly customized and specialized _Slice_ of the
Ubuntu distribution, which one could see as a block of stone from which
we can carve and extract the small and relevant parts we need to run
our applications.

It operates like a package manager, but for package slices, thus being
particularly useful for supporting developers in the creation of
smaller but equally functional container images.
```

Reply:
The first paragraph's from my discourse post. Others, just wrote it up.

Reply:
Updated in https://github.com/rebornplusplus/chisel/pull/6/commits/1c903e9084030821bcd69eb56c1d153ea693a4ec, thanks.